### ISSUE TITLE
InnoDB presently supports one FULLTEXT index creation at a time

### ISSUE THREAD

**Issue by chillu** [silverstripe/silverstripe-framework#9629]
With InnoDB (which is the new default engine in 4.6), you can't create two FULLTEXT indexes in the same dev/build. Since it doesn't create the first one either, this is a persistent error that can't be resolved without direct `ALTER TABLE` database queries.

Example of a failing definition on `DataObject`:

```php
    private static $indexes = [
        'PrioritySearchFields' => [
            'type' => 'fulltext',
            'columns' => ['Title'],
        ],
        'AllSearchFields' => [
            'type' => 'fulltext',
            'columns' => ['Title', 'Description'],
        ]
    ];
```

**Comment by maxime-rainville** [comment]
I presumed this limitation is "per table"? So if you have a fulltext index on a Company DataObject and a fulltext index on an Employee DataObject, your `dev/build` will complete normally.

**Comment by chillu** [comment]
That's correct, per table. But to be clear, this isn't resolved by running `dev/build` a second time - it's a persistent error until you remove the second index definition.
